custom/icds_reports/management/commands/sql_scripts/fix_past_data_part_1_4.sql

```diff
@@ -0,0 +1,78 @@
+UPDATE 
```

Comment:
you could have combined 1_4 to 1_7 as this is just rollups and filters like aggregation_level etc can be dynamic and passed while invoking. It will reduce the amount of code in this PR

Reply:
answered below